src/general/OptionsMenu.tscn

```diff
@@ -934,696 +961,705 @@ __meta__ = {
 margin_top = 3.0
 margin_right = 326.0
 margin_bottom = 28.0
 text = "AMOUNT_OF_AUTOSAVE_TO_KEEP"
 
-[node name="HSeparator2" type="HSeparator" parent="CenterContainer/VBoxContainer/PanelContainer/Misc/ScrollContainer/MarginContainer/VBoxContainer/HBoxContainer"]
+[node name="HSeparator2" type="HSeparator" parent="CenterContainer/VBoxContainer/VBoxContainer/VBoxContainer/PanelContainer/Misc/ScrollContainer/MarginContainer/VBoxContainer/HBoxContainer"]
 margin_left = 330.0
-margin_right = 437.0
+margin_right = 498.0
 margin_bottom = 31.0
 size_flags_horizontal = 3
 custom_styles/separator = SubResource( 4 )
 
-[node name="MaxAutoSaves" type="SpinBox" parent="CenterContainer/VBoxContainer/PanelContainer/Misc/ScrollContainer/MarginContainer/VBoxContainer/HBoxContainer"]
-margin_left = 441.0
-margin_right = 641.0
+[node name="MaxAutoSaves" type="SpinBox" parent="CenterContainer/VBoxContainer/VBoxContainer/VBoxContainer/PanelContainer/Misc/ScrollContainer/MarginContainer/VBoxContainer/HBoxContainer"]
+margin_left = 502.0
+margin_right = 702.0
 margin_bottom = 31.0
 rect_min_size = Vector2( 200, 25 )
 size_flags_vertical = 0
 min_value = 1.0
 max_value = 50.0
 value = 1.0
 
-[node name="HBoxContainer2" type="HBoxContainer" parent="CenterContainer/VBoxContainer/PanelContainer/Misc/ScrollContainer/MarginContainer/VBoxContainer"]
+[node name="HBoxContainer2" type="HBoxContainer" parent="CenterContainer/VBoxContainer/VBoxContainer/VBoxContainer/PanelContainer/Misc/ScrollContainer/MarginContainer/VBoxContainer"]
 margin_top = 146.0
-margin_right = 641.0
+margin_right = 702.0
 margin_bottom = 177.0
 
-[node name="Label" type="Label" parent="CenterContainer/VBoxContainer/PanelContainer/Misc/ScrollContainer/MarginContainer/VBoxContainer/HBoxContainer2"]
+[node name="Label" type="Label" parent="CenterContainer/VBoxContainer/VBoxContainer/VBoxContainer/PanelContainer/Misc/ScrollContainer/MarginContainer/VBoxContainer/HBoxContainer2"]
 margin_top = 3.0
 margin_right = 332.0
 margin_bottom = 28.0
 text = "AMOUNT_OF_QUICKSAVE_TO_KEEP"
 
-[node name="HSeparator2" type="HSeparator" parent="CenterContainer/VBoxContainer/PanelContainer/Misc/ScrollContainer/MarginContainer/VBoxContainer/HBoxContainer2"]
+[node name="HSeparator2" type="HSeparator" parent="CenterContainer/VBoxContainer/VBoxContainer/VBoxContainer/PanelContainer/Misc/ScrollContainer/MarginContainer/VBoxContainer/HBoxContainer2"]
 margin_left = 336.0
-margin_right = 437.0
+margin_right = 498.0
 margin_bottom = 31.0
 size_flags_horizontal = 3
 custom_styles/separator = SubResource( 4 )
 
-[node name="MaxQuickSaves" type="SpinBox" parent="CenterContainer/VBoxContainer/PanelContainer/Misc/ScrollContainer/MarginContainer/VBoxContainer/HBoxContainer2"]
-margin_left = 441.0
-margin_right = 641.0
+[node name="MaxQuickSaves" type="SpinBox" parent="CenterContainer/VBoxContainer/VBoxContainer/VBoxContainer/PanelContainer/Misc/ScrollContainer/MarginContainer/VBoxContainer/HBoxContainer2"]
```

Comment:
I can't navigate to this spin box with autosave disabled.

Reply:
When testing with a controller, I'm pretty sure I could, but the problem there was that the disabled spinbox doesn't have a separate focused style, so I couldn't see where the focus was.

Reply:
The sequence needed is (Autosave during the game) -> right -> down -> down. When the spin box is disabled, no GUI indicator is shown so it is unlikely that someone will perform this sequence correctly. It is a mixed issue because I think at least caret navigation should be prevented (potential Godot shortage)

Reply:
I see so the caret being invisible is an additional hurdle. I'll comment that on the issue 4010.